skbio/sequence/_dna.py

```diff
@@ -96,8 +97,9 @@ class DNA(IUPACSequence, NucleotideMixin):
 
     """
 
-    @classproperty
     @overrides(NucleotideMixin)
+    @stable(as_of="0.4.0")
+    @classproperty
```

Comment:
The correct ordering for this to work must be:

```python
@classproperty
@stable
@overrides
def foo():
    ...
```

Can you please update here and throughout?